bin/persephone.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+java -jar cli.jar $@
```

Comment:
We generally throw the `-server` flag into headless/cli stuff. I believe it is the default but it's convention for us, so I mention it. @cliffcaseyyet can probably decide if it needs to be here.

Reply:
I didn't include it because I wasn't sure what it did

Reply:
Well as I understand it, it omits all deps aside from the ones specified, yeah. A good example of why you might want this is logging libs complaining at compile time and stuff like that. If the build is good with `:replace-deps` I'd probably use that.

Reply:
As I understand it, Java  has two different optimizations of VMs, client and server. Server I think is the default, but the docs say this is "subject to change" so we set server.

Reply:
Keep this comment open so Cliff can respond, but I did add the `-server` flag in https://github.com/yetanalytics/project-persephone/pull/39/commits/99c4c3d441a944c53c441bac4f787c2918b92dfa